app/src/main/res/layout/activity_main.xml

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
-<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
I would say adding bottom nav menu should be one commit and adding vector assets should be another. I expected to see just vector assets added on this commit but saw you added the bottom nav menu. So this would be confusing to someone doing a code review.

Reply:
Oh, I added "created and added the bottom menu bar" in the comment description.
I think I forgot to add the commit name about the menubar
I will try to clearly name the commit next time